### ISSUE TITLE
TypeError: Cannot assign null to property OCA\Polls\Db\Log::$displayName

### ISSUE THREAD

**Issue by dirdi** [nextcloud/polls#2869]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

Cronjob is throwing stack traces occasionally.

### What did you expect, how polls should behave instead?

Cronjob should not crash.

### What steps does it need to replay this bug?

Run cronjob.

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

Updated from a minor version within same major version (i.e. 4.0.0 to 4.1.1)

### Affected polls version

5.0.3

### Which browser did you use, when experiencing the bug?

- [ ] Firefox
- [ ] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Other browser

N/A

### Which System did you use, when experiencing the bug?

- [ ] Windows
- [ ] Linux
- [ ] MacOs
- [ ] iOS
- [ ] Android
- [ ] Other/Don't know

### Other OS

N/A

### Add your browser log here

```shell
N/A
```

### Additional client environment information

N/A

### NC version

Nextcloud 26 (Nextcloud Hub 4)

### Other Nextcloud version

26.0.1

### PHP engine version

Other/Don't know

### Other PHP version

8.2.5

### Database engine

MariaDB

### Database Engine version or other Database

10.5.18

### Which user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```shell
TypeError: Cannot assign null to property OCA\Polls\Db\Log::$displayName of type string in /usr/www/users/user/domain.tld/lib/public/AppFramework/Db/Entity.php:136
Stack trace:
#0 /usr/www/users/user/domain.tld/lib/public/AppFramework/Db/Entity.php(167): OCP\AppFramework\Db\Entity->setter()
#1 /usr/www/users/user/domain.tld/lib/public/AppFramework/Db/Entity.php(75): OCP\AppFramework\Db\Entity->__call()
#2 /usr/www/users/user/domain.tld/lib/public/AppFramework/Db/QBMapper.php(321): OCP\AppFramework\Db\Entity::fromRow()
#3 /usr/www/users/user/domain.tld/lib/public/AppFramework/Db/QBMapper.php(339): OCP\AppFramework\Db\QBMapper->mapRowToEntity()
#4 /usr/www/users/user/domain.tld/apps/polls/lib/Db/LogMapper.php(81): OCP\AppFramework\Db\QBMapper->findEntities()
#5 /usr/www/users/user/domain.tld/apps/polls/lib/Service/MailService.php(65): OCA\Polls\Db\LogMapper->findUnprocessed()
#6 /usr/www/users/user/domain.tld/apps/polls/lib/Cron/NotificationCron.php(44): OCA\Polls\Service\MailService->sendNotifications()
#7 /usr/www/users/user/domain.tld/lib/public/BackgroundJob/Job.php(78): OCA\Polls\Cron\NotificationCron->run()
#8 /usr/www/users/user/domain.tld/lib/public/BackgroundJob/TimedJob.php(103): OCP\BackgroundJob\Job->start()
#9 /usr/www/users/user/domain.tld/lib/public/BackgroundJob/TimedJob.php(93): OCP\BackgroundJob\TimedJob->start()
#10 /usr/www/users/user/domain.tld/cron.php(152): OCP\BackgroundJob\TimedJob->execute()
#11 {main}
```

### Additional Information

This bug may be related to the one I reported before: https://github.com/nextcloud/polls/issues/2852#issuecomment-1516499457

**Comment by dartcafe** [comment]
Please make sure, that this log entry happened after the update to 5.0.3, because it got fixed in this version. 

I just checked that and I can't reproduce it

**Comment by dirdi** [comment]
Dear @dartcafe, I upgraded to 5.0.3 on 22.04. 06:46 and the Cronjob threw the cited error on 23.04. 04:35.

**Comment by dartcafe** [comment]
Thanks for clarification.

If possible, can you check line 67? It should contain the following definitions:

https://github.com/nextcloud/polls/blob/7885b1daebed74f24ab8ffeae0b3368d471b93ff/lib/Db/Log.php#L63-L68

**Comment by dirdi** [comment]
Nope, this is what `./app/polls/lib/Db/Log.php` looks like on my instance:
```protected int $pollId = 0;
protected int $created = 0;
protected int $processed = 0;
protected string $userId = '';
protected string $displayName = '';
protected string $messageId = '';
```

But admin panel states that version 5.0.3 of the app was installed?!?

**Comment by dirdi** [comment]
Just downloaded and checked the `.tar.gz` file of version 5.0.3 from here: https://github.com/nextcloud/polls/releases/tag/v5.0.3 The fix for this bug seems to no be included in the release.

**Comment by dartcafe** [comment]
grmpfl. 

Seems I messed up with the correct tagging. Since I fixed another bug today, I will release 5.0.4 later today. 

Thanks for your time and sorry for that confusion.

sorry @v1r0x  😁
